ts/Stock/Indicators/SMA/SMAIndicator.ts

```diff
@@ -15,7 +15,7 @@
  * */
 
 import type AxisType from '../../../Core/Axis/AxisType';
-import type Chart from '../../../Core/Chart/Chart';
+import Chart from '../../../Core/Chart/Chart.js';
```

Comment:
Could you move this away from type imports? Simply move them to the other imports